### PR TITLE
Finalised d.ASH Nav Mobile, Desktop and d.ASH Go User Guides

### PR DESCRIPTION
**Pull request by michaelpang294**
Checked by Brandon, Chow Yi, Hazel, Sean, Michael

### CODE REVIEW

File: README.md
Comment (anon-is-anonymus) [inline README.md:1]: Revert back to the change in master

File: docs/dash-nav/dash-nav.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+<div id="section2_1"></div> 
+
+### 2.1 Installation Guide
+Before installing, ensure that your mobile device meets the minimum system requirements. 
+
+| Minimum System Requirements                             | Recommended System Requirements                           |
+|--------------------------------------------------------|------------------------------------------------------------|
+| Android Tablet with support for OpenGL ES 3.2           | Android Tablet with support for OpenGL ES 3.2             |
+| Operating System: Android Version 13 \| Tiramisu \| API Level 33  | Operating System: Android Version 13 \| Tiramisu \| API Level 33  |
+| CPU: Qualcomm SM7125 Snapdragon 720G (8 nm)             | CPU: Snapdragon 888 Chip on Samsung Galaxy Tab S8        |
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: Add specifics of the chip and not just the model of tablet the chip is on

File: docs/dash-nav/dash-nav.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+| GPU: Adreno 618                                        | GPU: Adreno 730                                         |
+| RAM: 4 GB                                             | RAM: 8 GB                                              |
+
+<div id="section2_2"></div> 
+
+### 2.2 Your Account
+Login to d.ASH Nav with your given credentials.
+If you encounter any difficulties with logging in, please contact us.
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: No email for contact us

File: docs/dash-nav/dash-nav.md
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md:1]: Keep the references consistent. E.G: your table uses html href but the reference to offline mode in line 117 is in markdown format

Also what's the use of the divs for each section

File: docs/dash-nav/dash-nav.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+
+<div id="section5_3"></div>
+
+#### 5.3 Localising your robot 
+
+Before delving into the features in the planning tab, it is essential to establish an initial pose estimate to localise your robot. Localisation is a crucial step for the robot to accurately determine its position within the provided point cloud map. This process is a prerequisite for enabling autonomy on the robot.
+
+| Instructions |
+| ------- | 
+| <div class="instructions-div"><div class="instruction-circle">1</div><p style="margin: 0; margin-left: 5px;">Inside the options panel of the 'Ready' tab, press the 'Localise' button.</p></div><fonst size="3">![Localize Button](img/localize-button.png){style="margin-top: 10px; width: 1000px;"}<br><font size='2'>Here, you can also find your robot status and see information like the battery level, localisation and stance. |
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: font size not fonst size

File: docs/dash-nav/dash-nav.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+![Add New Route](img/add-new-route.jpg){style="width: 300px; margin-top: 10px;"}
+
+<div id="section6_1_1"></div>
+
+#### 6.1.1 Create routes by manually controlling of your robot
+From the list of routes on the right side, select the route that you have just created and press the 'Record Route' button boxed in red to record your route. 
+![Record Route Button](img/record-route-button.png){style="margin-top: 10px; width: 1000px;"}
+
+You will then be directed to the route recording view. Here, you will be able to use the joysticks and 'Add Point' buttons at the bottom to manually control your robot, creating your route. 
+You are advised to use a smaller recording interval (1 metre or smaller) when making turns. <br>
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: Why br

File: docs/dash-nav/dash-nav.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+![Record Route View GIF](img/record-route-view.gif){style="margin-top: 10px;"}
+
+After moving your robot, you should see a blue-coloured path being formed by the robot. This is the path your robots will follow when you run the route you are creating.
+
+Once you are finished with setting your route, press 'Save Recorded Route' at the bottom of the screen and you will be directed back to the routes page with your new route saved. Now, you will be able to make your robot run the route you have created. Refer to [Section 6.3](#section6_3).
+
+<div id="section6_1_2"></div>
+
+#### 6.1.2 Create routes by plotting waypoints on the map
+From the list of routes on the right side, select the route that you have just created and press this button boxed in red to record your route. <br>
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: Is the br necessary

File: docs/dash-nav/dash-nav.md [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+
+<div id="section6_1_3_2"></div>
+
+#### 6.1.3.2 Auto generation of scan points
+In this section, you can automatically generate BLK scanning points in your route based on the distance travelled by the robot.
+To use this, you must already have a route with waypoints. If you do not have one, refer to [Section 6.1.1](#section6_1_1) for a guide on recording your waypoints by manually controlling your robot or refer to [Section 6.1.2](#section6_1_2) for a guide on creating your route by manually plotting waypoints.
+
+To enable this feature, turn this option on and set a distance interval. If your set distance interval is two metres, the robot will run a BLK scan roughly every two metres travelled along your route.
+![Auto Generate Scan Points GIF](img/auto-generate-scan-points.gif){style="margin-top: 10px;"style="margin-top: 10px;"}
+The generated blinking points on the map is where your robot will apply a BLK scan.<br>
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: rm `<br>`

File: docs/dash-nav/dash-nav.md [hunk 7]
```diff
@@ -0,0 +1,387 @@
...
+#### 6.2 Editing or deleting your routes
+Navigate to the right-hand side, where you'll find a list of the routes you've created. To edit or delete any of these routes, press and hold on the desired route. This action will reveal options such as rename, copy, and delete at the bottom of the screen.
+![Edit Route GIF](img/edit-route.gif){style="margin-top: 10px;"}
+
+<div id="section6_3"></div>
+
+#### 6.3 Running your routes
+| Instructions |
+| ------- | 
+| <div class="instructions-div"><div class="instruction-circle">1</div><p style="margin: 0; margin-left: 5px;">Select the route that you would like to run.</p></div><fonst size="3"><font size='2'>You can find your routes in the routes section of the 'Plan' tab.|
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: font not fonst for line 361-364

File: docs/dash-nav/dash-nav.md [hunk 8]
```diff
@@ -0,0 +1,387 @@
...
+
+Before using offline mode, ensure that your map has already been downloaded in online mode. This is because there is no internet connection and the app is unable to fetch the maps from your cloud.  Refer to [Section 3.2](#section3_2) for a guide on loading your map into the application. 
+ 
+| Instructions |
+| ------- | 
+| <div class="instructions-div" id="step1"><div class="instruction-circle">1</div><p style="margin: 0; margin-left: 5px;">Set a local password to enable local robot login</p></div><fonst size="3"><font size='2'>Logged into the application (online mode), head into the 'Ready' tab and select 'Set Local Password' on the right of the screen.<br>![Set Local Password](img/set-local-password.jpg){style="width: 300px; margin-top: 10px;"}<br>For security purposes, you are advised to use a strong password that you can remember to prevent unauthorised access to your robot.<br>![Local Password Prompt](img/local-password-prompt.jpg){style="width: 300px; margin-top: 10px;"} |
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: fonst again

File: docs/dash-nav/dash-nav.md [hunk 9]
```diff
@@ -0,0 +1,387 @@
...
+| <div class="instructions-div"><div class="instruction-circle">4</div><p style="margin: 0; margin-left: 5px;">Connect to your robot.</p></div><fonst size="3"><font size='2'>On the left hand side of the main page, select on this button boxed in red to connect to your robot.<br>![Select Robot](img/select-robot.png){style="width: 300px; margin-top: 10px;"}<br>Your robot should appear now in the form of its IP address.<br>![Offline Mode Connect To Robot](img/offline-mode-connect-to-robot.jpg){style="width: 600px; margin-top: 10px;"}<br>Select your robot and enter the local password that you have previously set in [Step 1](#step1) to finally login.|
+
+You are now using offline mode. Now, you will be able to access the application's functionalities in locations without internet access.
+
+---
+
+That is all for d.ASH Nav!
+If you require any support, please feel free to contact us [here](mailto:hello@dconstruct.co) if you encounter any difficulties.
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: chg to .group

File: docs/dash-nav/dash-nav.md [hunk 10]
```diff
@@ -0,0 +1,387 @@
...
+
+This document was last updated on: 9 October 2023
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: chg to jus "Last updated: 9/10/2023"

File: docs/dash-pack/circlestyle.css
Comment (anon-is-anonymus) [inline docs/dash-pack/circlestyle.css:1]: Is this imported anywhere? If so leave a comment. If not just remove

File: docs/dash-pack/dash-go.md
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md:1]: Make sure the references are in markdown and make sure the ### are in the correct order/size (For this refer to xplorer docs)

File: docs/dash-pack/dash-go.md [hunk 11]
```diff
@@ -1,15 +1,103 @@
...
+### Introduction 
+
+Welcome to the d.ASH Go application user guide! This comprehensive guide will walk you through the process of installing the application and providing the necessary instructions for your map scanning process. To access the user guide for our application, please refer to <a href="#section2">Section 2</a> in this page. Otherwise, if you would like to scan maps on your web browser, please turn to <a href="#">Section 3</a>.
+![](img/d.ASHGo/dash-go.png){margin-top="-20"; width="1200"}
+
+### Application Functions 
+d.ASH Go is our visualisation system that allows you to see where you've mapped on the fly in just two easy steps! The point clouds are generated in real time, giving you a preview of what the final generated map would look like. 
+
+Before starting, an internet connection from your d.ASH Pack is required. If you are having trouble connecting to your d.ASH Pack's network, please contact us <a href="">here</a> for more information.
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Add email

File: docs/dash-pack/dash-go.md [hunk 12]
```diff
@@ -1,15 +1,103 @@
...
+
+### d.ASH Go Android Application
+
+#### Installation Guide
+Before installing, ensure that your mobile device meets the minimum system requirements. 
+
+Click <a href="https://play.google.com/store/apps/details?id=ai.dconstruct.dashpack">here</a> to install the d.ASH Go application on your Android device. Alternatively, you can enter the Google Play Store and search for our app. 
+
+
+#### Minimum System Requirements
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Add recommended req. See xplorer for ref

File: docs/dash-pack/dash-go.md [hunk 13]
```diff
@@ -1,15 +1,103 @@
...
+
+Android Tablet with support for OpenGL ES 3.2
+Operating System: Android Version 13 | Tiramisu | API Level 33 
+CPU: Qualcomm SM7125 Snapdragon 720G (8 nm)
+GPU: Adreno 618
+RAM: 4 GB
+
+![](img/d.ASHGo/google-play.png){width="600"}
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: SS desc wld be nice. Width is in px?

File: docs/dash-pack/dash-go.md [hunk 14]
```diff
@@ -1,15 +1,103 @@
...
+| <div class="instructions-div"><div class="instruction-circle">6</div><p style="margin: 0; margin-left: 5px;">Press and hold on the stop button to finish the map scan. </p></div><font size="3"><font size="2"> Well done! Now, you can preview your scan by selecting on your scan on the home page and pressing open scan at the bottom right. |
 
-## Requirements
+If there are major misalignments present in your scan, it is recommended to perform the scan once again. If not, you are good to go! 
+Use your map scan for further preprocessing in <a href="">d.ASH Xplorer</a> or upload it into the <a href="#">d.ASH Nav</a> application for use in robot automation.
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Broken links

File: docs/dash-pack/dash-go.md [hunk 15]
```diff
@@ -1,15 +1,103 @@
...
+
+#### Installation Guide
+Before installing, ensure that your mobile device meets the minimum system requirements. 
+
+Click <a href="https://play.google.com/store/apps/details?id=ai.dconstruct.dashpack">here</a> to install the d.ASH Go application on your Android device. Alternatively, you can enter the Google Play Store and search for our app. 
+
+
+#### Minimum System Requirements
+
+Android Tablet with support for OpenGL ES 3.2
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: check wif xq if d.ASH Go requires OpenGL ES 3.2 and not just OpenGL ES 3.0

File: docs/dash-pack/dash-go.md [hunk 16]
```diff
@@ -1,15 +1,103 @@
...
+| <div class="instructions-div"><div class="instruction-circle">6</div><p style="margin: 0; margin-left: 5px;">Press and hold on the stop button to finish the map scan. </p></div><font size="3"><font size="2"> Well done! Now, you can preview your scan by selecting on your scan on the home page and pressing open scan at the bottom right. |
 
-## Requirements
+If there are major misalignments present in your scan, it is recommended to perform the scan once again. If not, you are good to go! 
+Use your map scan for further preprocessing in <a href="">d.ASH Xplorer</a> or upload it into the <a href="#">d.ASH Nav</a> application for use in robot automation.
 
-- Android Tablet with support for OpenGL ES 3.0
-- Android Version 13 | Tiramisu | API Level 33
+### d.ASH Go Web Application
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Under d.ASH Pack not d.ASH Go

File: docs/dash-pack/dash-go.md [hunk 17]
```diff
@@ -1,15 +1,103 @@
...
+| <div class="instructions-div"><div class="instruction-circle">6</div><p style="margin: 0; margin-left: 5px;"> Press the stop scan button at the bottom to finish the maps scan. </p></div><font size="3">![](img/d.ASHGo/stop-scan-button.png){style="width: 200px; margin-top: 10px;" }<br>![](img/d.ASHGo/stop-scan-confirmation.png){style="width: 300px;"}<font size="2"><br> Well done! Your map scan has now been created. |
 
-1. Connect to your d.ASH Pack's Hotspot via the Android settings
-2. Click start recording and start your walk
-3. Enjoy your preview!
+Open <a href="#">d.ASH Xplorer</a> to the map scan that you have just created.
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Broken link

File: docs/dash-pack/dash-go.md [hunk 18]
```diff
@@ -1,15 +1,103 @@
...
 
+This document was last updated on: 10/09/2023
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: chg to jus "Last updated: 9/10/2023"

File: docs/dash-pack/dash-pack.md [hunk 19]
```diff
@@ -1,4 +1,4 @@
-# d.ASH Pack
+# **d.ASH Pack**
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-pack.md]: ???

File: docs/dash-pack/dash-pack.md [hunk 20]
```diff
@@ -13,7 +13,7 @@ You can view a short video overview of how **d.ASH Pack** integrates into your A
 - A Windows PC with d.ASH Xplorer
 - d.ASH Pack device
 
-*Users can plug in a 4G USB dongle into the side USB port of d.ASH Pack to remotely control it
+\*Users can plug in a 4G USB dongle into the side USB port of d.ASH Pack to remotely control it
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-pack.md]: is the \ needed?

File: docs/dash-pack/dash-pack.md [hunk 21]
```diff
@@ -27,14 +27,15 @@ Please ensure that others do not stand near the LiDAR while recording as this co
...
+
 The d.ASH Pack Wi-Fi network SSID and Password will be provided to you on the access panel of the d.ASH Pack itself.
 
 <!-- Insert image here -->
 
-Use this to connect to your d.ASH Pack and access the user interface by keying this IP Address (https://192.168.10.1/) into a web browser from any device of your choice. Users can check the status of their d.ASH Pack and start/stop recordings here.
+Use this to connect to your d.ASH Pack and access the user interface by keying this IP Address (http://192.168.10.1/) into a web browser from any device of your choice. Users can check the status of their d.ASH Pack and start/stop recordings here.
 
 Mobile interface:
 | ![Screenshot](img/d.ASHPack/d.ASHPack-userinterface.jpg){ align=center style="width:600px"} |
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-pack.md]: Swap to webapp pics

File: docs/stylesheets/extra.css [hunk 22]
```diff
@@ -5,4 +5,4 @@
 
 .header{
 	font-size: x-large;
-}
```
Comment (anon-is-anonymus) [inline docs/stylesheets/extra.css:8]: What this lol

File: docs/stylesheets/nav-style.css
Comment (anon-is-anonymus) [inline docs/stylesheets/nav-style.css:1]: Is this imported anywhere? If so comment if not remove

**Review by anon-is-anonymus — CHANGES_REQUESTED** [review]

File: docs/dash-nav/dash-nav.md [hunk 23]
```diff
@@ -85,15 +89,18 @@ Before installing, ensure that your mobile device meets the minimum system requi
...
 
+
+<!-- Click [here](https://play.google.com/store/apps/details?id=ai.dconstruct.dashnav) to install the d.ASH Nav application on your Android device. Alternatively, you can enter the Google Play Store and search for our app.  -->
+    
 <div id="section2_2"></div> 
 
 ### 2.2 Your Account
 Login to d.ASH Nav with your given credentials.
-If you encounter any difficulties with logging in, please contact us.
+If you encounter any difficulties with logging in, please contact us [here](mailto:hello@dconstruct.co).
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: shld be .group no .co

File: docs/dash-nav/dash-nav.md (hunk 9 above)
Comment (anon-is-anonymus) [reply docs/dash-nav/dash-nav.md]: @michaelpang294 ^

File: docs/dash-nav/dash-nav.md [hunk 24]
```diff
@@ -384,4 +391,4 @@ You are now using offline mode. Now, you will be able to access the application'
 That is all for d.ASH Nav!
 If you require any support, please feel free to contact us [here](mailto:hello@dconstruct.co) if you encounter any difficulties.
 
-This document was last updated on: 9 October 2023
+This document was last updated on: 10/09/2023
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: Follow the exact phrasing in the previous comment about this

File: docs/dash-pack/dash-go.md
Comment (anon-is-anonymus) [reply docs/dash-pack/dash-go.md:1]: @Seannyyboii still not done ^

**Review by anon-is-anonymus — CHANGES_REQUESTED** [review]

File: docs/dash-nav/dash-nav.md [hunk 25]
```diff
@@ -0,0 +1,359 @@
...
+The plan tab is used for preparing routes for your robots to follow and perform BLK scans.
+
+![Plan Tab View](img/plan-tab.png){style="margin-top: 10px; width: 1000px;"}
+
+#### 6.1 Creating your route
+Before starting, ensure that you have our localised your robot to the map. Refer to [Section 5.3](#section5_3) if you require a guide to localise your robot onto the map. 
+
+At the bottom right corner of your screen, select on the 'Add Route' button.
+
+![Add Route Button](img/add-route-button.jpg){style="width: 300px; margin-top: 10px'"}
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: ; not '

File: docs/dash-nav/dash-nav.md [hunk 26]
```diff
@@ -0,0 +1,359 @@
...
+Once you are finished with plotting your route, select the 'Done' button on the left of the mode buttons. Now, you will be able to make your robot run the route you have created. Refer to [Section 6.3](#section6_3).
+
+#### 6.1.3 BLK Scanning
+Before setting BLK scan points, you can configure your LiDAR configurations depending on your use case. 
+![BLK Scan View](img/blk-scan-view.png){style="width: 1000px; margin-top: 10px;"}
+
+#### 6.1.3.1 BLK Configurations
+##### LiDAR Scan Quality
+
+LiDAR Scan Quality affects the rotation speed of the LiDAR, with "Low" indicating a slower rate and "High" representing a faster rate. Adjusting this setting directly impacts the density of data points generated in your point cloud map.
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: Don't mention rotation speed only density

File: docs/dash-pack/dash-go.md [hunk 27]
```diff
@@ -1,15 +1,87 @@
...
+
+Joins spatial data on the fly, allowing users to generate long expressions in order to visualize the data of their surrounding environment. This can be used for transferring environmental data to autonomous robots for them to traverse through the environment.
+
+### d.ASH Go Android Application
+
+#### Installation Guide
+Before installing, ensure that your mobile device meets the minimum system requirements. 
+
+Click [here](https://play.google.com/store/apps/details?id=ai.dconstruct.dashpack) to install the d.ASH Go application on your Android device. Alternatively, you can enter the Google Play Store and search for our app. 
+
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: Requirements header missing

File: docs/dash-pack/dash-pack.md (hunk 19 above)
Comment (anon-is-anonymus) [reply docs/dash-pack/dash-pack.md]: Remove the **

File: docs/stylesheets/extra.css [hunk 28]
```diff
@@ -3,6 +3,6 @@
     margin: 0 auto;
 }
 
-.header{
-	font-size: x-large;
+.header {
+    font-size: 24px;
```
Comment (anon-is-anonymus) [inline docs/stylesheets/extra.css:7]: Check if this affected the rest of the sites

**Review by anon-is-anonymus — CHANGES_REQUESTED** [review]

File: docs/stylesheets/extra.css (hunk 28 above)
Comment (michaelpang294) [reply docs/stylesheets/extra.css:7]: does not affect

File: docs/dash-pack/dash-go.md [hunk 29]
```diff
@@ -1,15 +1,88 @@
...
+### d.ASH Go Android Application
+Before installing the application, ensure that your mobile device meets the minimum system requirements. 
+
+#### System Requirements
+
+| Minimum System Requirements                                       | Recommended System Requirements                                   |
+|-------------------------------------------------------------------|-------------------------------------------------------------------|
+| Android Tablet with support for OpenGL ES 3.2                     | Android Tablet with support for OpenGL ES 3.2                     |
+| Operating System: Android Version 13 \| Tiramisu \| API Level 33  | Operating System: Android Version 13 \| Tiramisu \| API Level 33  |
+| CPU: Qualcomm Snapdragon 720G                                     | CPU: Snapdragon 888 Chip                                          |
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-go.md]: chg to Qualcomm Snapdragon 888

File: docs/dash-nav/dash-nav.md [hunk 30]
```diff
@@ -0,0 +1,359 @@
...
+This comprehensive guide will walk you through setting up the application and give you detailed insights into every function of the d.ASH Nav Android Application. Additionally, you'll find step-by-step instructions on how to control your robots directly from your devices and create routes for waypoint automation.
+
+### 2.1 Installation Guide
+Before installing, ensure that your mobile device meets the minimum system requirements. 
+
+| Minimum System Requirements                                       | Recommended System Requirements                                   |
+|-------------------------------------------------------------------|-------------------------------------------------------------------|
+| Android Tablet with support for OpenGL ES 3.2                     | Android Tablet with support for OpenGL ES 3.2                     |
+| Operating System: Android Version 13 \| Tiramisu \| API Level 33  | Operating System: Android Version 13 \| Tiramisu \| API Level 33  |
+| CPU: Qualcomm Snapdragon 720G                                     | CPU: Snapdragon 888 Chip                                          |
```
Comment (anon-is-anonymus) [inline docs/dash-nav/dash-nav.md]: chg to Qualcomm Snapdragon 888

File: docs/dash-pack/dash-pack.md [hunk 31]
```diff
@@ -27,17 +27,19 @@ Please ensure that others do not stand near the LiDAR while recording as this co
...
+
 The d.ASH Pack Wi-Fi network SSID and Password will be provided to you on the access panel of the d.ASH Pack itself.
 
-<!-- Insert image here -->
+![Screenshot](img/d.ASHGo/dashpack-hatch.png){ align=center style="width:400px"} 
+
+Use this to connect to your d.ASH Pack and access the user interface by keying this IP Address (http://192.168.10.1/) into a web browser from any device of your choice. Users can check the status of their d.ASH Pack and start/stop recordings here.
 
-Use this to connect to your d.ASH Pack and access the user interface by keying this IP Address (https://192.168.10.1/) into a web browser from any device of your choice. Users can check the status of their d.ASH Pack and start/stop recordings here.
+**Mobile interface:**<br>
```
Comment (anon-is-anonymus) [inline docs/dash-pack/dash-pack.md]: no need `<br>`

**Review by anon-is-anonymus — CHANGES_REQUESTED** [review]

**Review by anon-is-anonymus — APPROVED** [review]

**Pull request merged**